GG/src/DropDownList.cpp

```diff
@@ -30,6 +30,7 @@
 #include <GG/Scroll.h>
 #include <GG/StyleFactory.h>
 #include <GG/WndEvent.h>
+#include "../util/Logger.h"
```

Comment:
Fixed.

I meant to remove it before I pushed this PR.

Keeping GG separate with a distinct responsibilities is good for code quality.

However, GG has no useful file logger with timestamps to see sequencing of events.  `std::cout` is a poor substitute.

All of the FreeOrion dependence in `Logger` is in LoggerWithOptionsDB.  Whenever a logger is helpful for debugging I include it and I've always intended to remove it.